Jenkinsfile

```diff
@@ -89,7 +89,7 @@ pipeline {
                     stage('Checkout code') {
                       steps {
                         dir('k-exercises') {
-                          git url: 'git@github.com:kframework/k-exercises.git'
+                          git url: 'git@github.com:kframework/k-exercises.git', branch: 'emptylists'
```

Comment:
Is this for the final version? I haven't seen it before.

Reply:
no, this is just for testing purposes.